python/python/glide/redis_client.py

```diff
@@ -380,8 +421,14 @@ async def _process_push(self, response: Response) -> None:
 
         try:
             self._pubsub_lock.acquire()
-            self._pending_push_notifications.append(response)
-            self._push_pubsub_messages_safe()
+            callback, context = self.config._get_pubsub_callback_and_context()
+            if callback:
+                pubsub_message = self._notification_to_pubsub_message_safe(response)
+                if pubsub_message:
+                    callback(pubsub_message, context)
```

Comment:
No, we should not hide the exceptions in the client code. This might lead to serious bugs - the clients is free to implement his error handling inside the callback